exonum_launcher/explorer.py

```diff
@@ -55,20 +55,26 @@ def get_instance_id(self, instance: Instance) -> Optional[int]:
 
         return None
 
+    def get_tx_status(self, tx_hash: str) -> bool:
```

Comment:
This name is a bit confusing. I'd name it `is_tx_succeed` or something like that, since it returns `bool`.